pkg/controller/machine/controller.go

```diff
@@ -95,9 +95,11 @@ type ReconcileMachine struct {
 // and what is in the Machine.Spec
 // +kubebuilder:rbac:groups=cluster.k8s.io,resources=machines,verbs=get;list;watch;create;update;patch;delete
 func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	// TODO(mvladev): Can context be passed from Kubebuilder?
```

Comment:
/cc @pwittrock @droot 

If it's too difficult to change the method signatures, maybe we could add context into the request so that you could pull it out via request.context() or something?